api-server/src/server/boot/settings.js

```diff
@@ -140,7 +140,7 @@ function updateMyPortfolio(...args) {
   )(...args);
 }
 
-// This API is reponsible for what campers decide to make public in their porfoile, and what is private.
```

Comment:
"profile" is also misspelled in this sentence.

Reply:
Fixed in https://github.com/freeCodeCamp/freeCodeCamp/pull/51246/commits/4d6f6ac2c79cb7cf674a5f911c204c1deef099a3